R/exercise.R

```diff
@@ -789,6 +792,27 @@ merge_options <- function(preserved_opts, inherited_opts, static_opts = list())
   opts[!grepl("^exercise", names(opts))]
 }
 
+local_restore_options <- function(.local_envir = parent.frame()) {
+  options <- options()
+  withr::defer(restore_options(options), envir = .local_envir)
```

Comment:
While the original code works, seeing a variable `options` quickly has me think that the value is the `options()` function. Let's change it to something other than `options`.

```suggestion
  opts <- options()
  withr::defer(restore_options(opts), envir = .local_envir)
```